content/blog/2020-08-26-fb-survey.Rmd

```diff
@@ -23,7 +23,7 @@ summary: |
   show you how to access the data, and highlight
   some of the exciting new directions that we're pursuing.
 acknowledgements: |
-  Logan Brooks and Katie Mazaitis
+  *Logan Brooks and Katie Mazaitis
```

Comment:
I extracted this change to a CSS style, thus we don't have to touch all those files